src/command/discord/mod.rs

```diff
@@ -0,0 +1,80 @@
+mod extensions;
+mod interaction;
+mod poistaesto;
+mod poistavahti;
+mod vahti;
+
+use async_trait::async_trait;
+use serenity::model::application::command;
+use serenity::model::application::interaction::Interaction;
+use serenity::model::gateway::GatewayIntents;
+use serenity::model::prelude::*;
+use serenity::prelude::*;
+
+use crate::command::Command;
+use crate::database::Database;
+use crate::error::Error;
+
+pub const NAME: &str = "discord";
```

Comment:
This could also be done with an id, we really don't need the name, at least not right now. It's just for identifying the commanders in the Torimies struct